maintainers/maintainer-list.nix

```
@@ -1303,6 +1303,12 @@
     matrix = "@a-n-n-a-l-e-e:matrix.org";
     name = "annalee";
   };
+  annoyingrains = {
+    email = "contact@annoyingrains.xyz";
+    github = "AnnoyingRains";
```

```suggestion
    github = "AnnoyingRain5";
```
please use your current github name